workday/12.1/modules/ROOT/pages/workday-connector-design-center.adoc

```diff
@@ -0,0 +1,68 @@
+= Workday - Design Center Configuration - Mule 4
+:page-aliases: connectors::workday/workday-design-center.adoc, connectors::workday/workday-connector-design-center.adoc
+
+Anypoint Design Center's Flow Designer enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Configure the target component for your app.
+
+For information on Design Center field values, see
+the xref:workday-reference.adoc[Workday Connector Reference].
+
+== Configure the Input Source Trigger
+
+A trigger starts your app. Workday Connector provides the New Object Trigger operation that starts your app when a new object is created in Workday. Alternatively, you can use the HTTP Listener operation to start your app from a browser
+or with a command such as `curl`. Another alternative is to use Scheduler to start your app on a timed schedule.
+
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*, and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started* and go to Flow Designer.
+. Click the trigger card.
+. If you are using Workday Connector as the input source trigger, search for the connector;
+otherwise, you can use HTTP or Scheduler.
+. Complete these fields when using the New Object Trigger operation:
```

Comment:
```suggestion
. Complete these fields when using the On Modified Objects trigger:
```
I used word trigger here since it was used here before, but I wonder if this should not be called "source" just like in workday/12.1/modules/ROOT/pages/workday-connector-studio.adoc